python/ray/experimental/channel/intra_process_channel.py

```diff
@@ -13,24 +12,23 @@ class IntraProcessChannel(ChannelInterface):
     IntraProcessChannel is a channel for communication between two tasks in the same
     worker process. It writes data directly to the worker's _SerializationContext
     and reads data from the _SerializationContext to avoid the serialization
-    overhead and the need for reading/writing from shared memory.
+    overhead and the need for reading/writing from shared memory. Note that if the
+    readers may mutate the data, users should deep copy the data themselves to avoid
+    side effects.
 
     Args:
-        actor_handle: The actor handle of the worker process.
+        num_readers: The number of readers that will read from this channel.
```

Comment:
Mention these can be the same method of the same actor reading multiple times

Reply:
Added comments in https://github.com/ray-project/ray/pull/46431/commits/ff34fd84e7f473d4c5e3b24e677da8f041ffc078